src/StatusQ/Components/StatusTagSelector.qml

```diff
@@ -198,11 +213,28 @@ Item {
                                                                                 2 * bgRect.anchors.margins +
                                                                                 userListView.anchors.topMargin + userListView.anchors.bottomMargin +
                                                                                 (userListView.model.count * 64): 0
+
+        function orderNamesModel() {
+            if(root.orderByReadonly) {
+                for(var i = 0; i < namesModel.count; i++) {
+                    var entry = namesModel.get(i)
+                    if(entry.isReadonly) {
+                        namesModel.move(i, 0, 1)
+                    }
+                }
+            }
+        }
     }
 
     implicitWidth: 448
     implicitHeight: (tagSelectorRect.height + d.suggestionContainerHeight) > root.maxHeight ? root.maxHeight : (tagSelectorRect.height + d.suggestionContainerHeight)
 
+    onOrderByReadonlyChanged: { d.orderNamesModel() }
+    Component.onCompleted: {
+        // Component initialization:
+        d.orderNamesModel()
```

Comment:
the TagSelector does a `sortModel(root.contactsModel);` on `Component.onCompleted` where is instantiated (eg. `CreateChatView`), could this be a problem in sync somehow?

Reply:
I think it is not a problem bc `orderNamesModel()` only manipulates `namesModel` order. It doesn't add neither remove elements. And `sortModel()` will use the current `namesModel` object to find elements but the elements themselfs will not change.